src/chrome/content/rules/TYPO3.xml

```diff
@@ -1,114 +1,25 @@
 <!--
-	For rules causing false/broken MCB, see TYPO3.org-falsemixed.xml.
-
-	Other TYPO3 rulesets:
-
-		- T3Blog.com.xml
-
-
 	Nonfunctional hosts in *typo3.org:
-
-		- demo ¹
-		- lists ²
-
-	¹ Dropped
-	² Refused
-
-
-	Problematic hosts in *typo3.org:
-
-		- flow ¹
-		- neos ²
-
-	¹ Mixed css from $self
-	² Mismatched
-
-
-	Insecure cookies are set for these domains and hosts:
-
-		- .typo3.org
-		- buzz.typo3.org
-		- forge.typo3.org
-		- .forum.typo3.org
-
-
-	Mixed content:
-
-		- css on flow from $self *
-		- Images flow from $self *
-		- favicon on buzz, flow from $self *
-		- Bug on ^, buzz, forum, wiki from piwik.typo3.org *
-
-	⁴ Secured by us
-
+		- lists
```

Comment:
api (404)
demo
government.v4.demo
flow (mismatch)
neos (mismatch)
styleguide
t3con0[7-9]
t3con10-frankfurt
t3con12de

Could you also document the reason why they are not working?